.github/workflows/build.yml

```diff
@@ -510,30 +510,81 @@ jobs:
-          fi
+          CRATE_VERSION=$(cargo read-manifest --manifest-path $BUILD_CRATE_MANIFEST |  jq -r '.version')
+          LAST_COMMIT_SHA=$(git rev-parse --short HEAD)
+          VERSION="$CRATE_VERSION-$LAST_COMMIT_SHA"
 
-          sed -i "s/\(<version>\)[^<>]*\(<\/version>\)/\1$VERSION\2/g" /interop/csharp/RadixDlt.RadixEngineToolkit.Native.nuspec
+          sed -i "s/\(<version>\)[^<>]*\(<\/version>\)/\1$VERSION\2/g" /interop/csharp/RadixDlt.RadixEngineToolkit.nuspec
 
           echo "Configured Version: $VERSION"
-      - name: Copy uniffi artifacts
-        run: 
+      - name: Copying UniFFI Bindings
+        run: |
+          # We copy the UniFFI bindings to the interop/csharp/generated/ directory
+          mkdir ./interop/csharp/generated
+          cp ./artifacts/extracted/UniFFI-Bindings/output/radix_engine_toolkit_uniffi.cs ./interop/csharp/generated
+
           #Copy uniffi c# binding to directory interop\csharp\generated\
           #Copy uniffi native libraries to platform specific directories in interop\csharp\native
+      - name: Copying Libraries
+        run: |
+          # We copy the UniFFI bindings to the interop/csharp/native directory
+          mkdir ./interop/csharp/native
+
+          mkdir ./interop/csharp/native/x86_64-unknown-linux-gnu
+          cp ./artifacts/extracted/x86_64-unknown-linux-gnu/libradix_engine_toolkit_uniffi.so ./interop/csharp/native/x86_64-unknown-linux-gnu
+          
+          mkdir ./interop/csharp/native/i686-unknown-linux-gnu
+          cp ./artifacts/extracted/i686-unknown-linux-gnu/libradix_engine_toolkit_uniffi.so ./interop/csharp/native/i686-unknown-linux-gnu
+          
+          mkdir ./interop/csharp/native/aarch64-unknown-linux-gnu
+          cp ./artifacts/extracted/aarch64-unknown-linux-gnu/libradix_engine_toolkit_uniffi.so ./interop/csharp/native/aarch64-unknown-linux-gnu
+          
+          mkdir ./interop/csharp/native/x86_64-apple-darwin
+          cp ./artifacts/extracted/x86_64-apple-darwin/libradix_engine_toolkit_uniffi.dylib ./interop/csharp/native/x86_64-apple-darwin
+          
+          mkdir ./interop/csharp/native/aarch64-apple-darwin
+          cp ./artifacts/extracted/aarch64-apple-darwin/libradix_engine_toolkit_uniffi.dylib ./interop/csharp/native/aarch64-apple-darwin
+          
+          mkdir ./interop/csharp/native/x86_64-pc-windows-gnu
+          cp ./artifacts/extracted/x86_64-pc-windows-gnu/radix_engine_toolkit_uniffi.dll ./interop/csharp/native/x86_64-pc-windows-gnu
+
+      - name: DEBUG
+        run: |
+          sudo apt-get install tree
+          tree interop
+
       - name: NuGet Pack
         working-directory: interop/csharp/
         run: dotnet pack --configuration Release --output .
       - name: Publish Packages
         working-directory: interop/csharp/
-        run: dotnet nuget push RadixDlt.RadixEngineToolkit.Native.*.nupkg --source https://api.nuget.org/v3/index.json --api-key ${{ secrets.NUGET_ORG_API_KEY }}
```

Comment:
We should not be publishing these to the "Native" package which used to just contain the binaries, we should publish them to the other package.

Reply:
We would not be separating them, everything (bindings + binaries) would live in one Nuget package, I just don't want it to have the name "Native" as that implies that this is a package of just binaries.